wordpress-mobile/WordPress-iOS
# [Gutenberg] Image Size networking (native)

Alternative to https://github.com/wordpress-mobile/WordPress-iOS/pull/12706

As a proof of concept, this PR implements the minimum necessary to make Image Size works.
In the future we might need to create a more generic api call with a dynamic request method (given from gutenberg). We also might need to handle other api version Paths.

Related PRs:
`gutenberg-mobile`: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1455

**To test:**
- **Run metro server on https://github.com/wordpress-mobile/gutenberg-mobile/pull/1455**
- Add a image block to gutenberg.
- Select an image from the WordPress media Library.
- Select settings and choose a different image setting.
- Change to HTML mode.
- Check that the image URL has changes according to the selected image size.

Update release notes:
